src/main/java/org/bahmni/mart/config/view/RegViewDefinition.java

```diff
@@ -71,6 +74,12 @@ private String createCoalesceQueries(List<String> columnNames, List<String> tabl
     private String getCoalesceQuery(List<String> tableNames, String columnName) {
         List<String> columnNames = tableNames.stream()
                 .map(tableName -> String.format("%s.%s", tableName, columnName)).collect(Collectors.toList());
+
+        if (OBS_DATETIME.equals(columnName) || DATE_CREATED.equals(columnName)) {
```

Comment:
@Mahitha1467 Please simplify the `if-else`. You don't need a **else** block after **return** statement

Reply:
@sumanmaity112 that is else-if and I need that condition